### ISSUE TITLE
Table of Contents Colours

### ISSUE THREAD

**Issue by en0on** [sumatrapdfreader/sumatrapdf#3810]
I noticed that in previous builds, the main window background colour was different from the table of contents (ie, the table of contents colour defaulted to the text's background colour, or image 1.) In the current pre-release build (v15797), the table of context is defaulting to the main window background (as shown in image 2). Is there a way to separate these values? i would like the table of contents to not be the main window background colour, especially because it makes it harder to differentiate the colours when reading. 

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/93896500/32f71241-7bce-4347-a66c-f2304865689f)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/93896500/76650e74-9587-4c85-a50c-38543d12740c)

**Comment by GitHubRulesOK** [comment]
This problem comes up intermittently with each significant change to one or the other of 3 :-)

The aim is to ensure canvas bookmarks and favourites have same background, but every so often they get updated at different rounds, so currently all is swell except theming interferes
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/baa78641-40dc-4ee5-9b32-b1d3b183ea83)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4adfab3f-0144-4a0f-ada1-d7c107d8e243)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6ddf8110-9973-4542-803a-ca4d401db013)

**Comment by alekssmitt** [comment]
YES, I noticed it right away too. The background color of the main window is different from the background color of the bookmarks window. It's very annoying. I had to roll back to the old one - 3.4.6

**Comment by GitHubRulesOK** [comment]
Closing as the behaviours have considerably changed now with different theming settings thus contrast is generically enhanced. Settings also include disabling the HOME tab if you don't want it!

![Image](https://github.com/user-attachments/assets/a2a25d2f-9ced-48b1-82eb-53c9c4d8d25a)
